Authentication/SignUp/ViewModel/SignupViewModel.swift

```diff
@@ -22,9 +22,13 @@ public protocol SignupViewModelType {
     
     var password: MutableProperty<String> { get }
     var passwordValidationErrors: AnyProperty<[String]> { get }
+    var showPassword: MutableProperty<Bool> { get }
```

Comment:
Now that I think about a little bit, using verb is better for method than properties. Which makes `passwordVisible` a better candidate.